target/dita/regions/regions.dita

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="links_1">
+ <title>
+  Regions
+ </title>
+ <body>
+  <imagemap>
+   <image href="./Content/Images/world-map.gif"/>
+   <area>
+    <shape>poly</shape>
+    <coords>
+     550,211,522,231,515,263,540,275,564,269,561,231,555,214
+    </coords>
+    <xref format="dita" href="./regions.dita"/>
+   </area>
+   <area>
+    <shape>poly</shape>
+    <coords>
+     540,326,611,308,681,326,711,286,659,128,618,139,588,199,572,273
+    </coords>
+    <xref format="dita" href="./regions.dita"/>
+   </area>
+   <area>
+    <shape>poly</shape>
+    <coords>
```

Comment:
actually @AbegaM - it would be easier to read if the `coords` elements were all on one line, too.